### PR TITLE
revisit outlook integration client code

### PR DESCRIPTION
**Pull request by amiturgman**
1. Remove xhr calls,  replacing with jQuery ajax calls
2. remove clientEnv dependency, reading document service url from the service

@cloudbeatsch please review and merge

### CODE REVIEW

**Comment by cloudbeatsch** [comment]
@amiturgman I updated getUserId to work with the new API (moved from POST with token in body to GET with token in header)

**Comment by cloudbeatsch** [comment]
@amiturgman the jquery call to the document service is not working because the request is formatted as a query string where the service expects a json body - we either change the request or the service - let me know what you prefer

**Comment by amiturgman** [comment]
@cloudbeatsch from what I can see, the request to the document service is sending the data in the body using a POST method. See the Functions.js file line 207- https://github.com/CatalystCode/ibera-office-integration/pull/13/files#diff-0c440cec4b556e4732f7ee16f5297113R207). 
But anyway, if you see  otherwise I agree it should be POST with the data in the body, I just couldn't test it since the service didn't work so I had to bypass it while testing hoping I wrote the code right for the actual http call.

File: static/Functions.js [hunk 1]
```diff
@@ -208,7 +208,7 @@ function processAttachments(isUpload, cb) {
           url: config.documentServiceUrl + "/api/Attachment",
           type: 'POST',
           contentType: "application/json; charset=utf-8",          
-          data: data,          
+          data: JSON.stringify(data),
```
Comment (amiturgman) [inline static/Functions.js]: Right... :)

Comment (cloudbeatsch) [reply static/Functions.js]: It works now ;-)

Comment (amiturgman) [reply static/Functions.js]: Cool! (Why are you working today?!)
This is what happen when I just take a quick look at my emails on the weekend... Sorry I drugged you into this... :)

Comment (cloudbeatsch) [reply static/Functions.js]: np - very bad weather today ;-)

Comment (amiturgman) [reply static/Functions.js]: ok. enjoy the rest of your weekend! :)
I see there's a small confilict, I'll resolve and merge this thing tomorrow...

Comment (cloudbeatsch) [reply static/Functions.js]: no hurry, the conflict comes from the change of the user api from POST to GET - should be straight forward - enjoy the weekend

**Pull request merged**